mantle/platform/qemu.go

```diff
@@ -654,6 +655,17 @@ func (disk *Disk) prepare(builder *QemuBuilder) error {
 			return err
 		}
 		qcow2Opts += fmt.Sprintf(",backing_file=%s,lazy_refcounts=on", backingFile)
+		format := disk.BackingFormat
+		if format == "" {
+			// QEMU 5 warns if format is omitted, let's do detection for the common case
+			// on our own.
+			if strings.HasSuffix(backingFile, "qcow2") {
+				format = "qcow2"
+			}
+		}
+		if format != "" {
+			qcow2Opts += fmt.Sprintf(",backing_fmt=%s", format)
+		}
```

Comment:
Should there be a final `else` where this can error out of `format` is still empty?

Reply:
Well...qemu still supports autodetection today, it just warns about it now.  And we rely on that in a whole lot of code paths.  Trying to make everything explicit would make sense but would be *way* more work.

This just quiets `cosa run` basically.